frontend/src/redux/reducers/popupLocationReducer.ts

```diff
@@ -5,14 +5,14 @@ import {
 } from 'redux/types/popupLocation';
 
 export const initialState: locationState = {
-  _id: ' aa',
+  _id: '',
   locationName: '',
   rating: {
-    likes: 0,
-    dislikes: 0
+    likes: [],
+    dislikes: []
```

Comment:
why likes is array?

Reply:
array with user id

Reply:
What options do you have to save likes and dislikes of a particular user? In addition, you need to look at whether the user has already liked or disliked and the best option is an email or id.